content/en/docs/concepts/configuration/secret.md

```diff
@@ -442,7 +442,7 @@ When a secret being already consumed in a volume is updated, projected keys are
 Kubelet is checking whether the mounted secret is fresh on every periodic sync.
 However, it is using its local cache for getting the current value of the Secret.
 The type of the cache is configurable using the  (`ConfigMapAndSecretChangeDetectionStrategy` field in
-[KubeletConfiguration struct](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go)).
+[KubeletConfiguration struct](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/pkg/kubelet/apis/config/types.go)).
 It can be either propagated via watch (default), ttl-based, or simply redirecting
```

Comment:
missing v1beta1?

Reply:
no v1beta1
u can see: 
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/config/types.go

Reply:
sorry, please change to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go

Reply:
updated.
Thanks!